src/Bicep.Core.Samples/Files/NestedResources_LF/main.sourcemap.json

```diff
@@ -0,0 +1,498 @@
+{
+  "12": {
+    "Item1": "main.bicep",
+    "Item2": 52
+  },
```

Comment:
Looking at this, we are 1-based rather than 0-based. 0-based feels like a more common and easier to handle format. Would it be possible to switch to 0-based?